bench/workgen/workgen.cpp

```diff
@@ -196,6 +201,7 @@ thread_tables_drop_workload(void *arg)
         runner->start_tables_drop(connection);
     } catch (WorkgenException &wge) {
         std::cerr << "Exception while dropping tables: " << wge._str << std::endl;
+        ASSERT(false);
```

Comment:
Have we checked that all of these exceptions we were not asserting before are okay to be fatal assertions now? We would have printed an error message earlier but continued running the workload and re-trying the failing operation. Now, we will exit the application. I think it should be okay to assert since this will uncover issues with the auxiliary operations.

Reply:
We would have continued the workload but the thread would exit which is not really useful, we don't want a workload to continue while threads are ko. I would prefer having those assertions to detect errors.